.github/2.0/cookbooks/Document.md

```diff
@@ -636,6 +638,100 @@ for evaluation_key, evaluation_score in d.evaluations.items():
  recall => recall at 10: 0.5
 ```
 
+### `GraphDocument`
+`GraphDocument` is a subclass of `Document`. It's a special type of `Document` that adds functionality to let you work with a `Document` as a `directed graph`.
+Chunks of the document represent the nodes of the graph. `GraphDocument` adds graph-specific attributes (`nodes`, `adjacency` list, `edge_features`,...) and operations (`add_node`, `remove_node`, `add_edge`, `remove_edge`,...)
+
+
+#### `GraphDocument` Additional Attributes
```

Comment:
Not sure, but don't we use sentence case for headers? Whatever it is, it is not consistent between this line and the next header "####`GraphDocument` methods".

Reply:
I applied sentence case for the rest of the headers and to this one too